vllm/v1/worker/tpu_model_runner.py

```diff
@@ -839,7 +836,7 @@ def dummy_run(
 
         with set_forward_context(attn_metadata, self.vllm_config, 0):
             assert self.model is not None
-            self.model(token_ids, position_ids, attn_metadata, kv_caches)
+            self.model(token_ids, position_ids, kv_caches)
```

Comment:
QQ: why do we need `kv_caches` here?

Reply:
This `self.model` is actually a `ModelWrapperV1`, which needs `kv_caches` in `forward` in order fiddle with the KV cache before executing the actual model.

Reply:
This `self.model` is actually a `ModelWrapperV1`, which needs `kv_caches` in `forward` in order fiddle with the KV cache before executing the actual model.